modules/ROOT/pages/Development/Satisfactory/Audio.adoc

```diff
@@ -7,14 +7,219 @@ This page is a work in progress.
 
 == Wwise
 
-TODO
-
 Coffee Stain uses the Wwise audio system to play sounds in Satisfactory.
 
-At the time of writing it is very difficult for modders to interface with Wwise systems,
-so you should use Unreal's built in sound system instead.
+To create new audio events to play in your mod, you will need the Wwise project set up and integrated into the SML project.
+
+=== Setting Up Audio Buses
+
+By default when using Wwise audio only the `Master` volume slider will work.
+
+You will need to manually create the different Audio Buses that link to the other sliders.
+
+TODO: Waiting for CSS to provide the correct audio bus names.
```

Comment:
What should people do in the mean time? Do we currently know any of the audio bus names? These, for example? https://discord.com/channels/555424930502541343/601030071221878784/1294145591709794379

Reply:
ATM, the master audio bus that Wwise has by default, correctly links to the in-game Master volume slider. This is currently the only way to change the volume of Wwise audio.

The variables linked to that Discord message are RTPC variables and not the names of the Audio Buses. I have tried a few different combinations of those just incase it worked. But this did not work. We will need to wait for CSS to provide the names of the Audio Buses.